.github/scripts/check-runtime.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```

Comment:
Maybe we should put this into the `pipeline-scripts` or `releng-scripts` repo, since it will be useful in the runtimes repo as well.

Reply:
Yes indeed, I would create a followup PR for the fellowship runtimes.
I think having this script in the `polkadot-sdk` repo does also make sense since downstream repos will already likely depend on the `polkadot-sdk` repo.